operator/config/overlays/community/manager_auth_proxy_patch.yaml

```diff
@@ -7,7 +7,7 @@ spec:
     spec:
       containers:
       - name: kube-rbac-proxy
-        image: gcr.io/kubebuilder/kube-rbac-proxy:v0.15.0
+        image: gcr.io/kubebuilder/kube-rbac-proxy:v0.16.0
```

Comment:
Why was this also updated? And why v0.16.0? It seems like v0.18.0 is the one aligned with k8s v1.30

Reply:
@JoaoBraveCoding Following the migration steps from `v1.33` up to `v1.37` I was instructed to update to `v0.16` here: https://sdk.operatorframework.io/docs/upgrading-sdk-version/v1.36.0/

You say `kube-rbac-proxy@v0.18.0` is using the `k8s@v1.30` client-go/api then we can update to this one too. I need to double check what version we issue on OCP v4.17 though.

Reply:
I see! Yes according to the release notes that seems to be the case https://github.com/brancz/kube-rbac-proxy/releases